kernel/src/process/process/init_proc.rs

```diff
@@ -13,11 +13,10 @@ use crate::{
     },
     prelude::*,
     process::{
-        Credentials, UserNamespace,
+        Credentials, ProgramToLoad, UserNamespace,
         posix_thread::{PosixThreadBuilder, ThreadName, allocate_posix_tid},
         process_table,
-        process_vm::new_vmar_and_map,
-        program_loader::ProgramToLoad,
+        process_vm::new_vmar_for_init,
```

Comment:
It's not `for_init` in the latest code anymore. Remove this method and use `Vmar::new(ProcessVm::new(...))` instead.